sections/featured-product.liquid

```diff
@@ -30,7 +30,7 @@
   <link id="ModelViewerOverride" rel="stylesheet" href="{{ 'component-model-viewer-ui.css' | asset_url }}" media="print" onload="this.media='all'">
 {%- endif -%}
 
-<section{% if section.settings.secondary_background %} class="background-secondary"{% endif %}>
+<section class="{% if section.settings.secondary_background %} background-secondary{% endif %} color-{{ section.settings.color_scheme }} gradient">
```

Comment:
Should be an if/else here as `.gradient` adds a background color that overwrites the one set in `background-secondary`. But still keep `color-{{ section.settings.color_scheme }}` at all time 🤔 
![](https://screenshot.click/22-01-x0fd1-bjefb.png)

Reply:
Gotcha - good point.